connector/keystone/types.go

```diff
@@ -0,0 +1,87 @@
+package keystone
+
+import (
+	"github.com/sirupsen/logrus"
+)
+
+type keystoneConnector struct {
```

Comment:
already in the keystone package, so this can just be called "connector"

Reply:
How about `conn` or plain `c`? Currently, other connectors use a similar name to what's introduced here (e.g. `microsoftConnector`), but I do agree that shorter is nicer, the context is clear here.

Reply:
Sure, it makes sense. I've changed to `conn`.